packages/gatsby-dev-cli/src/watch.js

```diff
@@ -17,7 +17,7 @@ function watch(root, packages, { scanOnce, quiet }) {
     const ignoreRegs = [
       /[\/\\]node_modules[\/\\]/i,
       /\.git/i,
-      new RegExp(`${p}[\\/]src[\\/]`, `i`),
+      new RegExp(`${p}[\\/\\\\]src[\\/\\\\]`, `i`),
```

Comment:
@jquense I think I get it now. So `\\` means `\` in a regex, but because the string itself also has a `\` escape character (for things like a newline `\n`), every regex `\` needs to be escaped in the string also. So `\\\\` is escaping the regex escape character and the regex target character.